telemetry_examples/src/main/java/com/newrelic/telemetry/examples/TelemetryClientExample.java

```diff
@@ -67,13 +85,24 @@ public static void main(String[] args) throws Exception {
 
     MetricBatch batch = metricBuffer.createBatch();
 
-    // The retrying sender uses the recommended techniques for responding to errors from the
+    // The TelemetryClient uses the recommended techniques for responding to errors from the
     // New Relic APIs. It uses a background thread to schedule the sending, handling retries
     // transparently.
-    sender.sendBatch(batch);
+    telemetryClient.sendBatch(batch);
+  }
 
-    // make sure to shutdown the sender, else the background Executor will stop the program from
-    // exiting.
-    sender.shutdown();
+  private static void sendSampleSpan(SpanBatchSender spanBatchSender, Attributes commonAttributes)
+      throws ResponseException {
+    Span sampleSpan =
+        Span.builder(UUID.randomUUID().toString())
+            .timestamp(System.currentTimeMillis())
+            .durationMs(150d)
+            .serviceName("Test Service")
+            .name("testSpan")
+            .build();
+    String traceId = UUID.randomUUID().toString();
+    SpanBatch spanBatch =
+        new SpanBatch(Collections.singleton(sampleSpan), commonAttributes, traceId);
+    spanBatchSender.sendBatch(spanBatch);
```

Comment:
I think this example should call through the `TelemetryClient instance` instead of the sender, right?

Reply:
oh mos def. let me fix